ImperatorToCk3Tests/ImperatorWorldTests/Characters/CharacterTests.cpp

```diff
@@ -0,0 +1,379 @@
+	input << "=\n";
+	input << "{\n";
+	input << "}";
+
+	const ImperatorWorld::Character theCharacter(input, 42);
+
+	ASSERT_TRUE(theCharacter.getSpouses().empty());
+}
+
+TEST(ImperatorWorld_CharacterTests, childrenCanBeSet)
+{
+	std::stringstream input;
+	input << "=\n";
+	input << "{\n";
+	input << "\tchildren = { 69 420 } ";
+	input << "}";
+
+	const ImperatorWorld::Character theCharacter(input, 42);
+
+	ASSERT_FALSE(theCharacter.getChildren().empty());
+	ASSERT_EQ(theCharacter.getChildren().begin()->first, 69);
+	ASSERT_EQ(theCharacter.getChildren().find(420)->first, 420);
+}
+
+TEST(ImperatorWorld_CharacterTests, childrenDefaultToEmpty)
+{
+	std::stringstream input;
+	input << "=\n";
+	input << "{\n";
+	input << "}";
+
+	const ImperatorWorld::Character theCharacter(input, 42);
+
+	ASSERT_TRUE(theCharacter.getChildren().empty());
+}
+
+TEST(ImperatorWorld_CharacterTests, motherCanBeSet)
+{
+	std::stringstream input;
+	input << "=\n";
+	input << "{\n";
+	input << "\tmother=123";
+	input << "}";
+
+	const ImperatorWorld::Character theCharacter(input, 42);
+
+	ASSERT_EQ(theCharacter.getMother().first, 123);
+}
+
+TEST(ImperatorWorld_CharacterTests, motherDefaultsToZero)
+{
+	std::stringstream input;
+	input << "=\n";
+	input << "{\n";
+	input << "}";
+
+	const ImperatorWorld::Character theCharacter(input, 42);
+
+	ASSERT_EQ(theCharacter.getMother().first, 0);
+}
+
+TEST(ImperatorWorld_CharacterTests, fatherCanBeSet)
+{
+	std::stringstream input;
+	input << "=\n";
+	input << "{\n";
+	input << "\tmother=123";
+	input << "}";
+
+	const ImperatorWorld::Character theCharacter(input, 42);
+
+	ASSERT_EQ(theCharacter.getMother().first, 123);
+}
+
+TEST(ImperatorWorld_CharacterTests, fatherDefaultsToZero)
+{
+	std::stringstream input;
+	input << "=\n";
+	input << "{\n";
+	input << "}";
+
+	const ImperatorWorld::Character theCharacter(input, 42);
+
+	ASSERT_EQ(theCharacter.getFather().first, 0);
+}
+
+TEST(ImperatorWorld_CharacterTests, familyCanBeSet)
+{
+	std::stringstream input;
+	input << "=\n";
+	input << "{\n";
+	input << "\tfamily=123";
+	input << "}";
+
+	const ImperatorWorld::Character theCharacter(input, 42);
+
+	ASSERT_EQ(theCharacter.getFamily().first, 123);
+}
+
+TEST(ImperatorWorld_CharacterTests, dynastyDefaultsToZero)
+{
+	std::stringstream input;
+	input << "=\n";
+	input << "{\n";
+	input << "}";
+
+	const ImperatorWorld::Character theCharacter(input, 42);
+
+	ASSERT_EQ(theCharacter.getFamily().first, 0);
+}
+TEST(ImperatorWorld_CharacterTests, wealthCanBeSet)
+{
+	std::stringstream input;
+	input << "=\n";
+	input << "{\n";
+	input << "wealth=\"420.5\"";
+	input << "}";
+
+	const ImperatorWorld::Character theCharacter(input, 42);
+
+	ASSERT_EQ(420.5, theCharacter.getWealth());
+}
+
+TEST(ImperatorWorld_CharacterTests, wealthDefaultsToZero)
+{
+	std::stringstream input;
+	input << "=\n";
+	input << "{\n";
+	input << "}";
+
+	const ImperatorWorld::Character theCharacter(input, 42);
+
+	ASSERT_EQ(0, theCharacter.getWealth());
```

Comment:
ASSERT_FALSE is also ok.

Reply:
I think the current one is more clear, let's keep true and false for bool.